kebechet/cli.py

```diff
@@ -88,7 +90,28 @@ def cli_run_results(origin, service, analysis_id):
 @click.option("-s", "--service", envvar="KEBECHET_SERVICE_NAME")
 def cli_run_url(url, service):
     """Run Kebechet by providing url to a git repository service."""
-    config.run_url(url, service)
+    config.run_url(url, service, None)
+
+
+@cli.command("run-webhook")
+@click.argument("web_payload", nargs=1)
+def cli_run_webhook(web_payload):
+    """Run Kebechet by providing a webhook payload."""
+    payload = None
+    if os.path.isfile(web_payload):
+        with open(web_payload) as f:
+            try:
+                payload = json.load(f)
+            except json.decoder.JSONDecodeError:
+                _LOGGER.error("Webhook file coudn't be parsed as a json.")
+    else:
+        # If the json is passed a string.
+        try:
+            payload = json.loads(web_payload)
+        except json.decoder.JSONDecodeError:
+            _LOGGER.error("Webhook payload coudn't be parsed.")
```

Comment:
Do not catch the exception here and above. Otherwise, process silently terminates with 0 return code

Reply:
Removed try catch :+1: